### PR TITLE
Added ARM64 support for generating ARM64 docker containers and added ARM64 support to Travis-CI

### LINKED ISSUE #288: Add aarch64 support to powerfulseal
Hi Team,

It will be very helpful if we have aarch64 support added in docker containers.

Currently, this project focuses on making amd64 docker containers. For including aarch64 support, I have just added libssl-dev, libffi-dev dependencies and used kubectl-aarch64 in the RUN clause of Dockerfile. This has successfully generated an aarch64 docker image. If this sounds feasible, I would like to raise a PR for including aarch64 support to powerfulseal, and add arm64 jobs in .travis.yml as well.

Dockerfile: https://github.com/odidev/powerfulseal/blob/master/build/Dockerfile
Travis report: https://travis-ci.com/github/odidev/powerfulseal

Please share your opinion on the same.

### PR DESCRIPTION
**Pull request by odidev**
Resolves #288 

1. ARM64 architecture has been added to .travis.yml file.
2. Installed libffi-dev and libssl-dev in the RUN clause of the Dockerfile for ARM64. libffi-dev and libssl-dev are required for installing pynacl package in ARM64.
3. Added a check in the RUN clause of the Dockerfile, for downloading kubectl binary according to the
host platform.

Signed-off-by: odidev <odidev@puresoftware.com>

### CODE REVIEW

**Comment by odidev** [comment]
Hi Team,

Can you please review this PR?

**Comment by seeker89** [comment]
Hi, sorry for taking the time. This looks good, we'll just need to handle the the publishing part of travis config, so that we don't try to publish the image twice with the same name for each architecture: https://github.com/bloomberg/powerfulseal/blob/ca4cbf3bb14607b5015474b54378625f0cf315bb/.travis.yml#L30-L44

I think for the pip, we'll just need to only trigger on one arch, but for docker, we'll need to modify the docker publish step to add postfix `-arm64` when architecture is set to ARM.

**Comment by odidev** [comment]
@seeker89 
Added an architecture check in Makefile for docker name and disabled pip deploy script for ARM64. Please let me know if any more changes required.

File: Makefile [hunk 1]
```diff
@@ -6,7 +6,13 @@ local_no_proxy ?= ""
...
 name ?= powerfulseal
 version ?= `python setup.py --version`
-tag = $(name):$(version)
+arch1=$(shell uname -m)
+ifeq ($(arch1),x86_64)
+  arch2=amd64
+else ifeq ($(arch1),aarch64)
+  arch2=arm64
+endif
+tag = $(name):$(version)-$(arch2)
```
Comment (seeker89) [inline Makefile:15]: Thanks for making this change!

This is looking good, but for backwards compatibility I'd prefer to keep the naming scheme unchanged, and only append `-arm64` ad the end when necessary.

```suggestion
arch=$(shell uname -m)
ifeq ($(arch),x86_64)
  tag = $(name):$(version)
else ifeq ($(arch1),aarch64)
  tag = $(name):$(version)-arm64
endif
```

**Comment by odidev** [comment]
@seeker89 
I have made suggested changes in the Makefile. Please suggest if anything else is required.

**Pull request merged**